moncli/entities/column_value/simple.py

```diff
@@ -1,15 +1,77 @@
 from moncli.error import ColumnValueError
 from moncli.entities.column_value.objects import PersonOrTeam
 from moncli import enums
-
+from datetime import datetime, timedelta
 from .base import SimpleNullValue, ComplexNullValue
 from .constants import SIMPLE_NULL_VALUE
 from ...error import ColumnValueError
+from moncli.config import DATE_FORMAT,TIME_FORMAT
+import pytz
 
 class DateValue(ComplexNullValue):
     """A date column value."""
-    pass
+    native_type = datetime
+    allow_casts = (int, str)
+    has_time = False
+
+    def _convert(self,value):
+        try:
+            new_time = datetime.strptime(value['time'], TIME_FORMAT)
+            new_date = datetime.strptime(value['date'], DATE_FORMAT)
+            self.has_time = True
+            new_date = pytz.timezone('UTC').localize(new_date)
+            new_date = new_date + timedelta(hours=new_time.hour, minutes=new_time.minute, seconds=new_time.second)
+            date_value = new_date.astimezone(datetime.now().astimezone().tzinfo) 
+            return date_value
+
+        except KeyError:
+            new_date = datetime.strptime(value['date'], DATE_FORMAT)
+            return new_date
         
+    def _cast(self,value):
+        if isinstance(value,int):
+            try: 
+                new_date_value =  datetime.fromtimestamp(value)
+                date=str(new_date_value.date())
```

Comment:
Here is where the problem likely begins.  Remember that once the value is converted from the API, we should always assume that it is local.  By running the _convert method below at line 38, it will assume that a local time is actually a UTC time and perform another (unnecessary) conversion.  I would suggest returning just the contents of line 34 as it already produces a localized datetime from the timestamp.  You don't need the rest...